### PR TITLE
[Users] Added usable fixtures for testing users to address issue #684

### LINKED ISSUE #684: [Users+Profiles] Create a common fixture file for consistent testing
I think as profiles, notifications, and friending become more complicated, it may make sense to create a common fixture file with users, notifications, and other model instances to simplify and standardize our testing practices. Additionally, as front-end starts to develop the look of the profiles, this will reduce our need to create multiple users each time to test edge cases and front-end can use the file as well to test different profile settings/attributes.

### PR DESCRIPTION
**Pull request by harriet-fisher**
To address issue #684 I adjusted existing fixture files for better testing procedures that align with our new user and user profile models. 

- User List Updated: Harriet, Holly, David, Rain, Jonathan Users
- Each user has a unique name, email, tokens, and moderator status
- Each user has a matching profile and a bio.
- Friend Relationships: Friendships are now represented using FriendInvitation objects (not a friends list).
Harriet <-> Holly (accepted)
Harriet <-> David (accepted)
Holly <-> Rain (accepted)
David <-> Jonathan (accepted)
Rain -> Jonathan (pending)

Removed old users, the friends field from profiles, and any unrelated or extra data.

### CODE REVIEW

**Review by davdma — CHANGES_REQUESTED** [review]
Hey @harriet-fisher most of the code looks good, but I just want to clear up some confusion before merging this - my PR #836 removed `display_name` as an attribute of the profile model in favor of using the `name` attribute in the user model. Can you just double check that the fixtures work with that?

File: src/chigame/users/fixtures/users_fixture.json [hunk 1]
```diff
@@ -25,10 +27,12 @@
     "is_staff": false,
     "is_active": true,
     "date_joined": "2023-10-30T03:21:40.210Z",
-    "name": "user1",
-    "email": "user1@example.com",
+    "name": "Holly",
```
Comment (hollyfwei) [inline src/chigame/users/fixtures/users_fixture.json:30]: :)

**Review by hollyfwei — APPROVED** [review]
Looks great, Harriet! Will be helpful for testing.

**Comment by harriet-fisher** [comment]
@davdma I did notice the changes and my second push to this PR removed the display names so it should be good to go!

**Review by jonathanbacon22 — APPROVED** [review]
Thank you for adding this! I see that David's concerns were addressed. LGTM!

**Review by davdma — APPROVED** [review]
Thanks for addressing my concerns Harriet! Everything looks good to me now!

**Review by cmarshall282 — APPROVED** [review]
Thanks for updating our fixture!
